nautobot_chatops/workers/nautobot.py

```diff
@@ -388,6 +388,14 @@ def get_interface_connections(dispatcher, filter_type, filter_value_1, filter_va
                 f'Unknown filter type "{filter_type}"',
             )  # command did not run to completion and therefore should not be logged
 
+        # Check on empty choice list, send an error back
+        if not choices:
+            dispatcher.send_markdown(
+                message=f"Unable to apply filter on '{filter_type}', please validate that data is present for a filter",
+                ephemeral=True
+            )
+            return False
```

Comment:
This might be worth recording as a STATUS_FAILED case, as it shouldn't be happening once you have a well-populated Nautobot instance?

Reply:
I was 50/50 on this one. I think that does make sense.